services/jobs/src/main/java/com/linkedin/openhouse/jobs/services/JobsRegistry.java

```diff
@@ -31,16 +31,19 @@ public JobLaunchConf createLaunchConf(String jobId, JobConf conf) {
       throw new JobEngineException(String.format("Job %s is not supported", type));
     }
     JobLaunchConf defaultConf = jobLaunchDefaultConfByType.get(type);
-    JobLaunchConf.JobLaunchConfBuilder builder = defaultConf.toBuilder();
     Map<String, String> propsMap = defaultConf.getSparkProperties();
     /*
     if properties has authTokenPath, read and set authToken as spark.sql.catalog.openhouse.auth-token
     in properties
     */
     if (authTokenPath != null) {
       propsMap.put("spark.sql.catalog.openhouse.auth-token", getToken(authTokenPath));
-      defaultConf.setSparkProperties(propsMap);
     }
+    // handle job properties defined in iceberg table.properties
```

Comment:
are we considering using table.properties to set this ? don't remember we ever discuss this.

Reply:
Artifact of older change. Deleted.